app/src/main/java/ch/epfl/favo/view/tabs/addFavor/FavorDetailView.java

```diff
@@ -104,6 +114,7 @@ private FavorStatus verifyFavorHasBeenAccepted(Favor favor) {
 
   private void acceptFavor() {
     currentFavor.setStatusIdToInt(FavorStatus.ACCEPTED);
```

Comment:
I noticed this on previous master but the status should be changed in the favorFuture.thenAccept, not before, because otherwise the favor fetched from the db will not be equal to the currentFavor and the favor will not be accepted because it thinks it was remotely changed. Maybe I can repair it on your branch?

Reply:
I think I've fixed that, if you want you can have a look and check it. Feel free to jump in and commit on my branch if you need.

Reply:
Ok I was wrong, I didn't fix it correctly. I'm going to fix it directly on the next PR about notifications since some other changes are needed to make the whole flow work correctly.